backend/tracim_backend/lib/core/user.py

```diff
@@ -267,7 +268,11 @@ def get_users_in_common_with_user_workspace(self, user_id: int) -> List[Tuple[An
         return (
             self._session.query(s2.user_id, User.username, User.display_name, s2.workspace_id)
             .join(s1, s1.workspace_id == s2.workspace_id)
```

Comment:
using .filter(s1.user_id == user_id) here should greatly reduce the request time

Reply:
Done, it works !